src/module.ts

```diff
@@ -511,8 +513,50 @@ export default defineNuxtModule<ModuleOptions>({
       ...contentContext as any
     }
 
+    // @nuxtjs/tailwindcss support
+    // @ts-ignore - Module might not exist
+    nuxt.hook('tailwindcss:config', (tailwindConfig) => {
+      tailwindConfig.content = tailwindConfig.content ?? []
+      tailwindConfig.content.push(`${nuxt.options.buildDir}/cache/content/parsed/**/*.md`)
```

Comment:
Important note: This path is highly liked to be changed for `.nuxt/cache` to differentiate between dev and prod.

Reply:
Design note: This integration could be in tailwind. It is tailwind that being extended by content not content by tailwind.

Reply:
Is there any util or option to get cache dir?

Reply:
Check https://github.com/nuxt/content/pull/1411#discussion_r940226190 for better solution Generally never depend on buildDir filesystem structure.

Reply:
> Design note: This integration could be in tailwind. It is tailwind that being extended by content not content by tailwind.

This integration moved to the content module (#1351) based on a discussion in the tailwind module.  https://github.com/nuxt-community/tailwindcss-module/pull/499#discussion_r918823965

Reply:
> Check https://github.com/nuxt/content/pull/1411#discussion_r940226190 for better solution Generally never depend on buildDir filesystem structure.

This implies double fetching remote sources even in dev mode, which had performance downsides for users in dev mode.

Reply:
> This implies double fetching remote sources even in dev mode, which had performance downsides for users in dev mode.

No it won't! You just readirect fs writes :)

> Design note: This integration could be in tailwind. It is tailwind that being extended by content not content by tailwind. (resolving to discuss in other comment)

> This integration moved to the content module (https://github.com/nuxt/content/pull/1351) based on a discussion in the tailwind module. https://github.com/nuxt-community/tailwindcss-module/pull/499#discussion_r918823965

Ah my bad. Let's do this later.